db/migrate/20171020215318_move_insurance_info_to_member.rb

```diff
@@ -0,0 +1,65 @@
+class MoveInsuranceInfoToMember < ActiveRecord::Migration[5.1]
+  def up
+    add_column :members, :is_insured, :boolean
+    add_column :members, :insurance_type, :string
+    add_column :medicaid_applications, :anyone_is_insured, :boolean
+
+    safety_assured do
+      execute <<~SQL
+          UPDATE medicaid_applications SET anyone_is_insured=insured;
+  
+          UPDATE members
+          SET 
+            is_insured=medicaid_applications.insured,
+            insurance_type=medicaid_applications.insurance_type
+          FROM (
+            SELECT
+              min(id) as id, benefit_application_id
+            FROM 
+              members
+            WHERE 
+              benefit_application_type = 'MedicaidApplication'
+            GROUP BY benefit_application_id
```

Comment:
This internal/sub query - actually probably with the larger query in question too - I think it'd be super beneficial to have a brief comment (above the `SELECT`) describing the what and why for the query. Where am I coming from with that request? I can get by with enough sql knowledge to figure things out but reading someone _else's_ queries can sometimes be a tall order. So, comments in more involved queries are especially helpful. I'll only speak for myself here but I don't think I'm alone(?)

Reply:
From what I can tell from the context around this, though - we're grouping members by the application, and returning only the "primary"/lowest ID'd member of that group. Correct?

Reply:
Yeah, I totally agree with the comment ... comment! I still have to step through it when I read, and I'm the one who wrote it. min is particularly tricky. You're right in your understanding, but let's make getting there a bit easier.